packages/perseus/src/widgets/numeric-input/numeric-input.class.tsx

```diff
@@ -35,7 +35,7 @@ export type NumericInputProps = ExternalProps & {
     rightAlign: NonNullable<ExternalProps["rightAlign"]>;
     apiOptions: NonNullable<ExternalProps["apiOptions"]>;
     coefficient: NonNullable<ExternalProps["coefficient"]>;
-    answerForms: NonNullable<ExternalProps["answerForms"]>;
+    answerForms: ReadonlyArray<PerseusNumericInputAnswerForm>;
```

Comment:
I was wondering, what does removing `answerForms` from the widget options but keeping it here accomplish?

Reply:
I didn't add either; `answerForms` on widget options was deprecated and unused, `answerForms` on answers existed already and is used.